massgov/mayflower-patternlab
# DP-8362 Update most/all Mayflower icons to use the Twig icon() function

Note:
Need to fix the link arrow display on home page banner in FF.  The style is not applied properly.

## Description

Replace the svg icon templates and the template calls with the icon() function.

Associated Drupal PR:
[DP-8362 Update most/all Mayflower icons to use the Twig icon() function #2128](https://github.com/massgov/mass/pull/2128)

## Related Issue / Ticket

- [DP-8362 Update most/all Mayflower icons to use the Twig icon() function](https://jira.state.ma.us/browse/DP-8362)

## Steps to Test


1. Go to any pages with icons.
2. Compare the page with one in `https://mayflower.digital.mass.gov/` and all icons are displayed as they are supposed to.
3. Check source for the icons for their markup.  You'll see 2 sets of `<svg>`.  The first one contains `<use>` as its child.  The second one has `<symbol>` for its child.
<img width="354" alt="mf-render" src="https://user-images.githubusercontent.com/9633303/38894569-960dbd26-425b-11e8-81c9-c3d013c39d5b.png">



## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
